src/main/java/pl/cleankod/ApplicationInitializer.java

```diff
@@ -64,13 +87,53 @@ FindAccountAndConvertCurrencyUseCase findAccountAndConvertCurrencyUseCase(
     }
 
     @Bean
-    AccountController accountController(FindAccountAndConvertCurrencyUseCase findAccountAndConvertCurrencyUseCase,
-                                        FindAccountUseCase findAccountUseCase) {
-        return new AccountController(findAccountAndConvertCurrencyUseCase, findAccountUseCase);
+    AccountController accountController(FindAccountService findAccountService) {
+        return new AccountController(findAccountService);
     }
 
     @Bean
     ExceptionHandlerAdvice exceptionHandlerAdvice() {
         return new ExceptionHandlerAdvice();
     }
+
+    @Bean
+    public SimpleModule singleValueObjectModule() {
+        SimpleModule module = new SimpleModule();
+        module.addSerializer(SingleValueObject.class, new JsonSerializer<>() {
+            @Override
+            public void serialize(
+                    SingleValueObject singleValueObject,
+                    JsonGenerator jsonGenerator,
+                    SerializerProvider serializerProvider
+            ) throws IOException {
+                jsonGenerator.writeObject(singleValueObject.value());
+            }
+        });
+        module.addDeserializer(Account.class, new AccountDeserializeService());
+        return module;
+    }
+
+    @Bean
+    public FindAccountService findAccountService(
+            FindAccountAndConvertCurrencyUseCase findAccountAndConvertCurrencyUseCase,
+            FindAccountUseCase findAccountUseCase
+    ) {
+        return new FindAccountService(findAccountAndConvertCurrencyUseCase, findAccountUseCase);
+    }
+
+    @Bean
+    public SimpleCacheManager cacheManager() {
+        SimpleCacheManager cacheManager = new SimpleCacheManager();
+        List<Cache> caches = new ArrayList<>();
+        caches.add(cacheBean().getObject());
```

Comment:
Why not just inject the cache as a parameter. The `ConcurrentMapCacheFactoryBean` would be triggered and would provide the underlying object.